dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/program/function/D2RelationshipCount.java

```diff
@@ -80,6 +80,8 @@ public Object getSql( ExprContext ctx, CommonExpressionVisitor visitor )
 
         return "(select count(*) from relationship r" + relationshipIdConstraint +
             " join relationshipitem rifrom on rifrom.relationshipid = r.relationshipid" +
-            " join trackedentityinstance tei on rifrom.trackedentityinstanceid = tei.trackedentityinstanceid and tei.uid = ax.tei)";
+            " join trackedentityinstance tei on rifrom.trackedentityinstanceid = tei.trackedentityinstanceid and tei.uid = ax.tei"
+            +
```

Comment:
Just cosmetic: Looks like spotless may have put the + on a new line. You could put the string continuation on the same line or, maybe better yet, just make a long string on the previous line including the where clause (like in ProgramIndicatorServiceD2FunctionTest.)